pom.xml

```diff
@@ -212,11 +233,20 @@
     </dependency>
 
     <!-- plugin dependencies -->
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>plain-credentials</artifactId>
+      <version>1.5</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>ssh-credentials</artifactId>
+      <version>1.13</version>
```

Comment:
This is not strictly necessary but when I run `mvn hpi:run` without this the configuration as code support plugin fails to load and leaves an error in the terminal. I don't know why that happens as everything CasC specific should remain scoped to testing. I'll see what I can do to suppress that warning without including this.

Reply:
These dependencies are needed because JCasC does not do plugin installation.  
You should scope these dependencies as either optional or test.
That should be handled by Jenkins Core or inside the POM.
See https://issues.jenkins-ci.org/browse/JENKINS-53767

Reply:
this shouldn't be needed anymore on latest JCasc